src/components/SideBar/SideBarComponents/CasesModal/index.js

```diff
@@ -195,8 +195,8 @@ const FacetModal = ({ closeModal, type, ...modalProps }) => {
         const fileItems = [];
         fileItemArray.map((item) => {
           if (!_.isEmpty(item)) {
-            item.replace('\r', '');
-            fileItems.push(item);
+            const newItem = item.replace('\r', '');
+            fileItems.push(newItem);
```

Comment:
no need of a new variable?

Reply:
Yes, we are using to remove unnecessary \r from uploaded CSV/txt files

Reply:
fileItems.push(item.replace('\r', ''));